db/migrate/20151029052305_create_messages.rb

```diff
@@ -1,10 +1,11 @@
 class CreateMessages < ActiveRecord::Migration
```

Comment:
yep.  any time you create a migration to update your database, you can simply run `rake db:migrate`.  It will update your schema automatically without forcing you to reset/rebuild your database.

Here are some tips:

`rake db:drop` - drop your current database
`rake db:create` - create a database with config data from `config/database.yml`
`rake db:migrate` - run all pending migrations to update your database schema
`rake db:migrate:reset` - reset your database and run migrations.  this is a pseudo shortcut for `rake db:drop db:create db:migrate`